src/main/java/uk/gov/hmcts/reform/blobrouter/util/TimeZones.java

```diff
@@ -1,8 +1,11 @@
 package uk.gov.hmcts.reform.blobrouter.util;
 
+import java.time.ZoneId;
+
 public final class TimeZones {
 
     public static final String EUROPE_LONDON = "Europe/London";
+    public static final ZoneId EUROPE_LONDON_ZONE_ID = ZoneId.of(EUROPE_LONDON);
```

Comment:
We'll probably have no more than this one time zone. How about moving it to DateTimeUtils, then? Also, as `EUROPE_LONDON` isn't referenced from anywhere else, probably best to use the value directly when creating zone id.

Reply:
Sorry, didn't notice this class already existed.

Reply:
Yes, and DateTimeUtils exist only in test modules